compiler/daml-lf-ast/src/DA/Daml/LF/Ast/Version.hs

```diff
@@ -186,7 +186,6 @@ featureExperimental = Feature
 foreverCppFlags :: [T.Text]
 foreverCppFlags =
     [ "DAML_NAT_SYN"
-    , "DAML_NAT_TYPE_ERASURE"
```

Comment:
I think we want to keep it: the reason we have these flags is in (the admittedly unlikely) case someone was relying on these CPP flags being set in their daml file. That way we don't break backwards source compatibility.

Reply:
This feature was never made stable, so nobody should rely on it.